builder/bundles.go

```diff
@@ -305,6 +309,7 @@ func queryRpmFullPath(packageCmd []string, pkgName string, repo string, repos ma
 	)
 	queryStringRpm = append(queryStringRpm, pkgName)
 	outBuf, _ := helpers.RunCommandOutputEnv(queryStringRpm[0], queryStringRpm[1:], []string{"LC_ALL=en_US.UTF-8"})
```

Comment:
Just to ensure we have all the info for debugging, lets catch the error returned from `helpers.RunCommandOutputEnv(queryStringRpm[0], queryStringRpm[1:], []string{"LC_ALL=en_US.UTF-8"})` and log it inside `if outBuf.String() == "" {`. E.g.
```
outBuf, err := helpers.RunCommandOutputEnv(queryStringRpm[0], queryStringRpm[1:], []string{"LC_ALL=en_US.UTF-8"})
log.Debug(log.Dnf, outBuf.String())
if outBuf.String() == "" {
                log.Debug(log.Dnf, err.Error())
		return "", fmt.Errorf("rpm not found for pkg: %s", pkgName)
	}
```